### PR TITLE
Add German HOWTO & CODEOFCONDUCT file

### PR DESCRIPTION
**Pull request by Squirkyy**
## What does this PR do?
Adds ONE HOWTO file (German) + ONE CODEOFCONDUCT file (german) and changes NINE HOWTO files
### Description
I added a German Translation for the HOWTO file and added the option to switch to German in all the other files.
I also added the German Translation for the Code of Conduct.
### Why is this valuable (or not)?
It helps the German learning community.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
Resolves #5684, Locks #5684

File: HOWTO-de.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+* [Github Hallo Welt](https://guides.github.com/activities/hello-world/) *(auf Englisch)*
+* [Youtube - Tutorial Github für Anfänger](https://www.youtube.com/watch?v=0fKg7e37bQE) *(auf Englisch)*
+* [Youtube - So forkst du ein Github-Repo und sendest einen Pull-Request](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(auf Englisch)*
+* [Youtube - Github Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo) *(auf Englisch)*
+
+
+Habe keine Angst eine Frage zu stellen; jeder muss irgendwo anfangen und den ersten Pull Request machen. Du könntest der 1000. sein.
+
+Selbst wenn du ein erfahrener Open-Source-Mitwirkender sind, gibt es Dinge, die dich ins Straucheln bringen könnten. Sobald du deinen PR eingereicht hast, führt GitHub Actions einen Linter aus und findet oft kleine Probleme von Absätzen oder Alphabetisierung. Wenn du eine grüne Schaltfläche erhältst, ist alles zur Überprüfung bereit, aber wenn das nicht so ist, klick unter dem fehlgeschlagenen Häkchen auf "Details", um herauszufinden, was dem Linter nicht gefallen hat. Behebe das Problem und füge deinen PR einen Commit hinzu.
```
Comment (davorpa) [inline HOWTO-de.md:15]: It satisfy #5644

**Review by davorpa — APPROVED** [review]
### Ready to merge :fireworks: :tada:

Let's see if this time we get it

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**